<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtTtValueLimitsWidget</class>
 <widget class="QWidget" name="mdtTtValueLimitsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>43</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lbMinusInfinity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-&amp;infin;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbLeftFailZone">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Left fail zone</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbLeftLimitBottomValue">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbLeftLimitRange">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Left limit range</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbLeftLimitTopValue">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbOkZone">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Ok range</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbRightLimitBottomValue">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbRightLimitRange">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Right limit range</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbRightLimitTopValue">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbRightFailZone">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Right fail zone</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbPlusInfinity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;+âˆž&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
